config/development.json

```
@@ -83,6 +83,7 @@
 		"jetpack/features-section/jetpack": true,
 		"jetpack/features-section/simple": true,
 		"jetpack/magic-link-signup": true,
+		"jetpack/onboarding-standalone-installation-v1": true,
```

Let us name it something that makes the purpose clearer

```suggestion
		"jetpack/update-standalone-plugin-onboarding-v1": true,
```

---

or

```suggestion
		"jetpack/standalone-plugin-onboarding-update-v1": true,
```

---

I have renamed the feature flag to 'jetpack/standalone-plugin-onboarding-update-v1'  which describes the new feature more clearly.